documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,150 @@
+      1. Qualification for an event must be based only on results from the same event.
+      2. Qualification for an event must be based on one type of result only (i.e. average/mean or single)
+         1. Exception: in case of using average/mean results, single results must be used to break ties.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+   4. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
+      1. The registration requirements must be available in English.
+      2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. Each registration period must be announced publicly on the WCA website at least 48 hours before it starts.
+         1. The competition's WCA Delegate(s) and/or organizer(s) may register for the competition prior to the first registration period, however their registration(s) must not be approved until the first registration period has started.
+      2. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
+   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+7. Events
+   1. Each event must have a planned number of rounds.
+      1. Each round of each event must have a planned format and time limit.
+      2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
+      3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
+   2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
+      1. Other events may only be held unofficially.
+   3. If the main event of the competition is not the 3x3x3 Cube, it should be publicly mentioned in the competition's information.
+8. Schedule
+   1. The schedule must be published on the WCA website using the WCA schedule tool.
+   2. The schedule should be manageable by the competition's WCA Delegate(s) and adhere to the WCA quality standards.
```

Comment:
"WCA quality standards" is mentioned here and in the "first" 8.2.1, but what are those? Such an expression requires a definition in my view.

Reply:
This term is not new. These standards are also mentioned in the WQAC's Motion:
https://www.worldcubeassociation.org/documents/motions/10.2017.5%20-%20Quality%20Assurance%20Committee.pdf

I'm okay with keeping the term here and adding a definition to the Motions, but I don't think the definition belongs in this document.

Reply:
I agree that an addition to the definitions should cover this off.

Reply:
I agree that this is not the place for the definition, but I'm not sure if the Definitions Motion is either... given how such "quality standards" have evolved so much, especially in the last couple of years, I'm afraid we'll need to be updating that motion continuously :confused:

Reply:
Hm, maybe this could be solved by requiring WQAC to come up with a "WCA Quality standards Policy" document. This might be overdue either way. 
The definition for "WCA quality standards" in the motions could be a cross-reference similar to the definition of "Regulations". At this point, it might also make sense to add a similar cross-reference defintion for "WCA competition requirements".

Reply:
Yeah, that makes sense. We can discuss this on an email thread, it's time to merge this PR up! :slightly_smiling_face: